projects/packages/videopress/src/client/block-editor/blocks/video/edit.tsx

```diff
@@ -158,28 +159,13 @@ export default function VideoPressEdit( {
 	const { filename, private_enabled_for_site: privateEnabledForSite } = videoData;
 
 	// Get video preview status.
-	const defaultPreview = { html: null, scripts: [], width: null, height: null };
-	const { preview, isRequestingEmbedPreview } = useSelect(
-		select => {
-			if ( ! videoPressUrl ) {
-				return {
-					preview: defaultPreview,
-					isRequestingEmbedPreview: false,
-				};
-			}
-
-			return {
-				preview: select( coreStore ).getEmbedPreview( videoPressUrl ) || defaultPreview,
-				isRequestingEmbedPreview:
-					select( coreStore ).isRequestingEmbedPreview( videoPressUrl ) || false,
-			};
-		},
-		[ videoPressUrl ]
-	);
-
-	// Pick video properties from preview.
-	const { html: previewHtml, scripts, width: previewWidth, height: previewHeight } = preview;
-
+	const { preview, isRequestingEmbedPreview }: UsePreviewResult = usePreview( videoPressUrl );
```

Comment:
Nitpick: We don't need to set the return type of `usePreview` as it's automatically inferred.

```suggestion
	const { preview, isRequestingEmbedPreview } = usePreview( videoPressUrl );
```